rust/minimal-counter-dapp/Makefile

```
@@ -0,0 +1,41 @@
+NAME = minimal_rust_dapp
+
+.PHONY: all
+all: build
+
+.PHONY: node_modules
+.SILENT: node_modules
+node_modules:
+	npm install
+
+.PHONY: build
+.SILENT: build
+build: node_modules
+	dfx canister create --all
+	dfx build
+
+.PHONY: install
+.SILENT: install
+install: build
+	dfx canister install --all
+
+.PHONY: upgrade
+.SILENT: upgrade
+upgrade: build
+	dfx canister install --all --mode=upgrade
+
+.PHONY: test
+.SILENT: test
+test: install
+	dfx canister call minimal_rust_dapp increment \
+		| grep '(1 : nat)' && echo 'PASS'
+	dfx canister call minimal_rust_dapp get \
+		| grep '(1 : nat)' && echo 'PASS'
+	dfx canister call minimal_rust_dapp reset \
+		| grep '(0 : nat)' && echo 'PASS'
```

This will fix the CI failure
```suggestion
		| grep '(1 : nat64)' && echo 'PASS'
	dfx canister call minimal_rust_dapp get \
		| grep '(1 : nat64)' && echo 'PASS'
	dfx canister call minimal_rust_dapp reset \
		| grep '(0 : nat64)' && echo 'PASS'
```